lib/soap/request.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Soap.Request do
+  defstruct [:method, :url, :body, :headers, :options]
+  @moduledoc """
```

Comment:
The documentation should be on the first line in the module

Reply:
Done.